assets/keymaps/default-macos.json

```diff
@@ -3,10 +3,14 @@
   {
     "bindings": {
       "up": "menu::SelectPrev",
+      "shift-tab": "menu::SelectPrev",
+      "home": "menu::SelectFirst",
       "pageup": "menu::SelectFirst",
       "shift-pageup": "menu::SelectFirst",
       "ctrl-p": "menu::SelectPrev",
       "down": "menu::SelectNext",
+      "tab": "menu::SelectNext",
+      "end": "menu::SelectLast",
```

Comment:
Just to be safe: are these idiomatic on macOS?

Reply:
no idea. I haven't been using macOS for any significant period of time. I believe it works this way on Windows, so my assumption it should work this way on macOS.

Reply:
I can confirm that the tab binding is!